### PR TITLE
Donations backend integration

### PR DESCRIPTION
**Pull request by zarvox**
Sits atop #277.  Requires that sandstorm-io/prod#12 go live before merge/deploy.

### CODE REVIEW

File: _layouts/page.html [hunk 1]
```diff
@@ -139,5 +155,60 @@
...
+    <!-- Donation form integration -->
+    <script src="https://checkout.stripe.com/checkout.js"></script>
+    <script>
+    var amountCents = 0;
+    var handler = StripeCheckout.configure({
+      key: "pk_live_bkrz8iFkfEeXDsdIqzDNdSkH",
+      image: "https://sandstorm.io/images/sandstorm-purplecircle.png",
+      locale: "auto",
+      token: function(token) {
+        console.log(token);
```
Comment (paulproteus) [inline _layouts/page.html]: I _guess_ if `console` isn't defined this will crash. I'll block on this.

**Comment by zarvox** [comment]
This is deployed now, so once the caches are invalidated 50% of viewers should see some donation bars.

**Comment by ocdtrekkie** [comment]
What if I want to see the donation bars but can't? (I actually don't know if we're past the cache time yet anyways.)

**Comment by zarvox** [comment]
@ocdtrekkie Try an incognito window, which will get a new experiment ID assigned (which will have a 50% chance of being in the experimental condition).

**Pull request merged**